ui/lib/pki/addon/routes/roles/create.js

```diff
@@ -17,7 +17,13 @@ export default class PkiRolesCreateRoute extends Route {
     const backend = this.secretMountPath.currentPath;
     return hash({
       role: this.store.createRecord('pki/role', { backend }),
-      issuers: this.store.query('pki/issuer', { backend }),
+      issuers: this.store.query('pki/issuer', { backend }).catch((err) => {
```

Comment:
🤔 I'm wondering what we want the user experience for the role create form to be here. Handling it this way means the form still renders, but the issuer dropdown will be empty. I suppose we don't have to worry about `403` here because the LIST endpoint is unauthenticated for issuers, right?

If we're okay with the role create form rendering, then we probably want to have the dropdown for the issuer input change to just a regular text input. 

Alternately, I think this could be addressed by using the `<SearchSelect>` in the form instead. I know there was discussion and we previously decided against it but it provides a nice UX because that component handles querying the api and rendering a different fallback component if there are any errors. Plus we could then remove the query from this route all together. 

Probably doesn't need to be addressed in this ticket since the role form isn't visbile until issuers are created, but I think it's nice to have all of our ducks in a row on the off chance a user directly navigates to the URL. 🦆 🦆 🦆

Reply:
I like the idea of falling back on a regular text input since that was the original mock-up! I'll update :)

Reply:
I'm going to make a separate ticket for this, @hellobontempo

Reply:
Great idea!

Reply:
I decided to add that fix to this PR. It will now fall back to an input field if issuers don't exist!